oio/account/client.py

```diff
@@ -148,7 +148,7 @@ def account_update(self, account, metadata, to_delete, **kwargs):
 
     def container_list(self, account, limit=None, marker=None,
                        end_marker=None, prefix=None, delimiter=None,
-                       **kwargs):
+                       buckets=False, **kwargs):
```

Comment:
Since it is not documented, this parameter lets me think buckets **won't** be listed by default.
I suggest:
- renaming this parameter so it tells what it does;
- documenting it.

Reply:
I've add a small doc on it.
But there is not many names that fit this options. Maybe `listing_buckets`.

Reply:
`s3_buckets_only`? `buckets_only`?